### ISSUE TITLE
Number of sample nodes under an edge

### ISSUE THREAD

**Issue by savitakartik** [tskit-dev/tsbrowse#148]
On the Edges tab, we could show the number of descendants for an edge, in the way that we do for mutations. (@fbaumdicker, do correct me if I have not understood your suggestion correctly!) 

As @jeromekelleher pointed out, this should be doable but not too trivial as the sets of sample nodes in the 'sub-clades' subtended by an edge might vary along the edge. 

On the UI side, we could either change line thickness (not sure if this would show up nicely in the datashaded plot but could do when we have less than a certain number of edges in the frame). Or we could display this information on click of an edge.

**Comment by fbaumdicker** [comment]
Yes, that was what I was thinking about. Changing the color shading instead of thickness might also work.
However, I did not think of the issue of varying sample nodes along the same edge. Maybe a mean value works for the start.
